x/fee/client/cli/query.go

```diff
@@ -76,7 +76,7 @@ func GetFungibleTokenFeeSetting(cdc *codec.Codec) *cobra.Command {
 
 			res, _, err := cliCtx.QueryWithData(fmt.Sprintf("custom/%s/%s/%s", "token", fungible.QueryTokenData, symbol), nil)
 			if err != nil {
-				fmt.Printf("This fungible token does not exist\n")
+				fmt.Printf("This fungible token does not exist: %s\n", symbol)
```

Comment:
@githubckgoh1439 "This" is still there.

Reply:
Fixed under this: 
https://github.com/maxonrow/maxonrow-go/pull/200/commits/28a37e076e1f52dda17d3bf53755b2ab281bc3e7